docs/core/tools/dotnet-sln.md

```diff
@@ -37,15 +37,13 @@ The `dotnet sln` command provides a convenient way to add, remove, and list proj
 
 `add <PROJECT> ...`
 
-`add **/*`
-
 Adds a project or multiple projects to the solution file. [Globbing patterns](https://en.wikipedia.org/wiki/Glob_(programming)) are supported on Unix/Linux based terminals.
 
 `remove <PROJECT> ...`
 
-`remove **/*`
+`remove **/*.csproj`
 
-Remove a project or multiple projects from the solution file. [Globbing patterns](https://en.wikipedia.org/wiki/Glob_(programming)) are supported on Unix/Linux based terminals.
+Removes a C# project or multiple C# projects from the solution file. [Globbing patterns](https://en.wikipedia.org/wiki/Glob_(programming)) are supported on Unix/Linux based terminals.
```

Comment:
Should we keep 'Removes' but drop 'C#'?
The command also removes F# projects too.